v1/ao/internal/reporter/reporter_udp.go

```diff
@@ -57,16 +59,23 @@ func (r *udpReporter) report(ctx *oboeContext, e *event) error {
 }
 
 // Shutdown closes the UDP reporter TODO: not supported
-func (r *udpReporter) Shutdown() error {
+func (r *udpReporter) Shutdown(ctx context.Context) error {
 	// return r.conn.Close()
 	return nil
```

Comment:
I would return an `not implemented error` from Shutdown function to indicate to the caller that is not implemented.

Reply:
Sounds good. Fixed :)